desk/lib/bolt.hoon

```diff
@@ -63,35 +60,35 @@
     ?>  =(src.bowl our.bowl)
     =/  =bean  !<(bean vase) 
     ?-  -.bean
-        %toggle-which
-      =.  which  switch.bean
-      :-  ~[(emit which/b+which)]
-          this 
+        %toggle-white
+      =.  white  switch.bean
+      :_  this
+      ~[(emit:json white/b+white)]
     ::
         %toggle-kids
       =.  kids  switch.bean
-      :-  ~[(emit kids/b+kids)]
-          this
+      :_  this
+      ~[(emit:json kids/b+kids)]
     ::
         %add-white
-        =.  users.whitelist  (~(uni in users.whitelist) users.bean)
-        :_  this           
-            ~[(emit add-users-white/(ships:util:json users.whitelist))]
+      =.  whitelist  (~(uni in whitelist) users.bean)
+      :_  this           
+      ~[(emit:json add-users-white/(ships:util:json whitelist))]
     ::
         %add-black
-      =.  users.blacklist  (~(uni in users.blacklist) users.bean)
+      =.  blacklist  (~(uni in blacklist) users.bean)
       :_  this           
-          ~[(emit add-users-black/(ships:util:json users.blacklist))]
+      ~[(emit:json add-users-black/(ships:util:json blacklist))]
     ::
         %remove-white
-      =.  users.whitelist  (~(dif in users.whitelist) users.bean) 
+      =.  whitelist  (~(dif in whitelist) users.bean) 
       :_  this
-          ~[(emit remove-users-white/(ships:util:json users.whitelist))]
-      ::
```

Comment:
Align the colcol with the wuthep to clearly show that one case is ending and another is beginning.